deeppavlov/dataset_iterators/dstc2_dialog_iterator.py

```diff
@@ -16,11 +16,11 @@
 from overrides import overrides
 
 from deeppavlov.core.common.registry import register
-from deeppavlov.core.data.dataset import Dataset
+from deeppavlov.core.data.dataset_iterator import BasicDatasetIterator
 
 
-@register('dialog_dataset')
-class DialogDataset(Dataset):
+@register('dstc2_dialog_iterator')
+class Dstc2DialogDatasetIterator(BasicDatasetIterator):
```

Comment:
The `DialogDataset`was intended to work not only with DSTC2 data, but any dialog data (babi, maluuba, etc.). So it preferably should be named `DialogDatasetIterator` or simular.

Reply:
Ok, done